h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -86,13 +106,19 @@ export default function AnnotationCard({
         const isConflictError =
           e instanceof APIError && e.response?.status === 409;
         const error = isConflictError
-          ? 'The annotation has been updated since this page was loaded'
-          : 'An error occurred updating the moderation status';
+          ? 'The annotation has been updated since this page was loaded. Review the details and try again.'
+          : 'An error occurred updating the moderation status.';
```

Comment:
Separate to the changes in this PR, the presentation of this message relies on color alone to indicate that something went wrong. Per our usual accessibility practices, we ought to have an icon or other non-color cue. Also when a reload happens, that's more of a warning rather than an error. Updating the presentation can be done in a follow-up PR.

Reply:
I have updated the logic to show three possible messages:

1. An unknown error occurred
2. A conflict occurred and we have loaded the annotation again
3. A conflict occurred but loading the annotation again failed

In a follow-up PR, among other improvements, I will make 2 be represented as a warning instead of an error.